modules/network/src/main/java/org/apache/ignite/internal/network/direct/DirectMessageWriter.java

```diff
@@ -39,6 +40,9 @@
  * Message writer implementation.
  */
 public class DirectMessageWriter implements MessageWriter {
+    /** Empty array-based byte buffer. Not read-only. */
+    public static final ByteBuffer EMPTY_BUFFER = ByteBuffer.wrap(ArrayUtils.BYTE_EMPTY_ARRAY);
```

Comment:
Suggestion: Maybe we can rename it `EMPTY_BYTE_ARRAY`?

Reply:
Sure